portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-cell-validators/generate-total-fees-accrued-currency-error.js

```diff
@@ -0,0 +1,32 @@
+const validator = require('validator');
+const { UTILISATION_REPORT_HEADERS } = require('../../../../constants');
+
+const generateTotalFeesAccruedCurrencyError = (csvDataRow) => {
```

Comment:
Seeing as you've done it in all the other files you added, could you add some JSDocs? It would be especially helpful here to interpret the difference between the null and `object` return type